recipe/meta.yaml

```
@@ -1,39 +1,52 @@
 {% set name = "geomet" %}
-{% set version = "1.0.0" %}
+{% set version = "1.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 0020a4426469934fb58f541cdc23f27c0c0fbbb3d003ee2cb76bb7ffa96a7506
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: be5c7b8268071bdb146ce16576402b6f436cb793ab210336fe01e2ab39978cac
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - geomet = geomet.tool:cli
+  skip: true  # [py<36]
```

```suggestion
  skip: true  # [py<37]
```

https://github.com/geomet/geomet/blob/1.1.0/pyproject.toml#L14

---

added!